locksmith/src/controllers/userController.ts

```diff
@@ -0,0 +1,40 @@
+import UserOperations = require('../operations/userOperations')
+import { Request, Response } from 'express-serve-static-core'
+
+namespace UserController {
+  export async function createUser(req: Request, res: Response): Promise<any> {
+    let user = req.body.user
+
+    try {
+      if (user) {
+        let creationStatus: Boolean = await UserOperations.createUser(
+          { emailAddress: user.emailAddress,
+          publicKey: user.publicKey,
+          passwordEncryptedPrivateKey: user.passwordEncryptedPrivateKey,
+          recoveryPhrase: user.recoveryPhrase}
+        )
+
+        let status = creationStatus ? 200 : 400
+        return res.sendStatus(status)
+      }
+    } catch (e) {
+      return res.sendStatus(400)
```

Comment:
500 would be more appropriate here? Also, I think we should log this?

Reply:
Can add the logging. Not sure about this being a 500. https://twitter.com/stevelosh/status/372740571749572610?lang=en 

lol